tests/unit/modules/docker_test.py

```diff
@@ -65,7 +65,7 @@ def test_ps_with_filters(self):
                 all=True,
                 filters={'label': 'KEY'})
 
-    @skipIf(_docker_py_version() > 0, 'docker-py needs to be installed for this test to run')
+    @skipIf(_docker_py_version() > (0,), 'docker-py needs to be installed for this test to run')
```

Comment:
Dam, I rebased and lost the jenkins build links.

Lets see what Jenkins says now.